### DISCUSSION TITLE
using ear trumpet in vr

### DISCUSSION THREAD

**Discussion by spark-thecat** [File-New-Project/EarTrumpet#1117] [Ideas]
hello,
    sometimes I like to use my computer in vr,  for watching movies or just chat in discord. 
    i've noticed that everytime I scroll it triggers a volume change.  the overlay I use to do the desktops things while in vr  is triggering mouse inputs that goes on the desktop. 
    i've noticed that ear trumpet has a mouse hover and scroll functionality which i like but I wish sometimes it could be disabled.  it seams that the desktops  overlay in vr emits the mouse wheel events and ear trumpet picks those up.

might be just me having those issue ;p

have a good day

**Comment by riverar** [comment]
@Spark010 Interesting, can you open an issue with this? Also be sure to mention what software you're using to access the desktop. I can then install it and try it here!

**Reply by spark-thecat** [reply]
will do :3
